ironfish/src/migrations/data/000-template/schemaNew.ts

```diff
@@ -0,0 +1,97 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { PUBLIC_ADDRESS_LENGTH } from '@ironfish/rust-nodejs'
+import bufio from 'bufio'
+import { IDatabase, IDatabaseEncoding, IDatabaseStore, StringEncoding } from '../../../storage'
```

Comment:
Should we make the template assuming more than 1 table will be modified? That's more of a pain but more of a scalable example.

Reply:
I think this is a good idea! I'll change the example into something contrived with more than 1 table read and more than 1 table write.